common/model-views.cpp

```diff
@@ -992,6 +994,22 @@ namespace rs2
             << "/" << (long long)this;
         populate_options(options_metadata, ss.str().c_str(), this, s, &_options_invalidated, error_message);
 
+        if (dev.supports(RS2_CAMERA_INFO_PHYSICAL_PORT) && dev.supports(RS2_CAMERA_INFO_PRODUCT_LINE))
+        {
+            std::string product = dev.get_info(RS2_CAMERA_INFO_PRODUCT_LINE);
+            std::string id = dev.get_info(RS2_CAMERA_INFO_PHYSICAL_PORT);
+            bool has_metadata = rs2::metadata_helper::instance().is_enabled(id) ||
+                !(product == "D400" || product == "SR300" || product == "L500");
```

Comment:
Can we abstract these product names into the metadata_helper class, such that we can simply ask:
```
auto & metadata = rs2::metadata_helper::instance();
bool has_metadata = metadata.is_enabled( dev );
```

That way, the product names the have metadata will be local to the helper class and this information is contained centrally.